### PR TITLE
Use Event Store to show faucet's deploys status.

### PR DESCRIPTION
**Pull request by zie1ony**
### Overview

Faucet: use event store to show deploy status.
Event store: better logs.

### CODE REVIEW

File: packages/ui/src/containers/FaucetContainer.ts [hunk 1]
```diff
@@ -61,15 +61,12 @@ export class FaucetContainer {
...
-        req.deployInfo!.execution_results.length === 0;
+      const needsUpdate = typeof req.deployInfo === 'undefined';
 
       if (needsUpdate) {
         anyNeededUpdate = true;
-        const info = await this.errors.withCapture(
-          this.tryGetDeployInfo(req.deployHashBase16)
-        );
+        const info = await this.tryGetDeployInfo(req.deployHashBase16);
+        console.log(info);
```
Comment (AbnerZheng) [inline packages/ui/src/containers/FaucetContainer.ts]: Useless log?

**Review by AbnerZheng — APPROVED** [review]
LGTM.

File: packages/ui/src/containers/FaucetContainer.ts (hunk 1 above)
Comment (zie1ony) [reply packages/ui/src/containers/FaucetContainer.ts]: done

**Pull request merged**